### PR TITLE
Update vsphere operations guide page

### PR DESCRIPTION
**Pull request by harsh-px**
Signed-off-by: Harsh Desai <harsh@portworx.com>

### CODE REVIEW

File: content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md [hunk 1]
```diff
@@ -58,111 +81,32 @@ The [Prometheus and Grafana on Kubernetes](/portworx-install-with-kubernetes/ope
...
+Increasing capacity falls into 2 primary buckets:
 
-    ```text
-    pxctl service maintenance --exit
-    ```
+### a. Increase storage capacity for a storage pool
 
-    Wait for about 2 minutes until `pxctl status` says *PX is operational*.
-10. List the Portworx pools again and you will now see the pool has the expanded size.
+As storage usage increases, you will need to expand the Portworx storage pools.
```
[the remark this answers is not in the document]
Comment (harsh-px) [reply content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md]: This doc aims to be a vsphere focused operations guide. Expanding storage pools is just part of it.

We can link the more detailed page you referenced above in this PR. Can you find a good place?

**Comment by harsh-px** [comment]
Thanks for the changes @nathan-px . I've merged all of them.

File: content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md [hunk 2]
```diff
@@ -58,113 +81,35 @@ The [Prometheus and Grafana on Kubernetes](/portworx-install-with-kubernetes/ope
...
-    ```text
-    pxctl service pool show
-    ```
-10. Resize the pool to account for the expanded VMDK.
+1. Identify the Portworx storage pool you want to expand by [listing them](#listing-storage-pools-created-by-portworx). 
+2. Enter the `pxctl service pool expand` command specifying the following:
 
+     * UUID: The UUID of the pool you want to expand. This is found the **POOL** column when you listed the pools.
+     * SIZE: The minimum new required size of the storage pool in GiB.
+     * OPERATION-TYPE: The type of disk operation that will be performed to expand the pool. You have 2 options
```
Comment (nathan-px) [inline content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md]:
```suggestion
     * OPERATION-TYPE: The type of disk operation that will be performed to expand the pool. You have 2 options:
```

File: content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md [hunk 3]
```diff
@@ -58,113 +81,35 @@ The [Prometheus and Grafana on Kubernetes](/portworx-install-with-kubernetes/ope
...
-10. Resize the pool to account for the expanded VMDK.
+1. Identify the Portworx storage pool you want to expand by [listing them](#listing-storage-pools-created-by-portworx). 
+2. Enter the `pxctl service pool expand` command specifying the following:
 
+     * UUID: The UUID of the pool you want to expand. This is found the **POOL** column when you listed the pools.
+     * SIZE: The minimum new required size of the storage pool in GiB.
+     * OPERATION-TYPE: The type of disk operation that will be performed to expand the pool. You have 2 options 
+         * **resize-disk**: Portworx will resize existing VMDKs in the storage pool. This is the recommended operation as this does not require data movement on the backing drives in the pool.
+         * **add-disk**: Portworx will create one or more VMDKs based on the required new size of the pool. After the drive is added to the pool, Portworx will rebalance the volumes onto the new VMDKs. Hence this operation can take a while.
     ```text
```
Comment (nathan-px) [inline content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md]:
```suggestion

    ```text
```

File: content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md [hunk 4]
```diff
@@ -58,113 +81,35 @@ The [Prometheus and Grafana on Kubernetes](/portworx-install-with-kubernetes/ope
...
-    ```
-10. Resize the pool to account for the expanded VMDK.
+1. Identify the Portworx storage pool you want to expand by [listing them](#listing-storage-pools-created-by-portworx). 
+2. Enter the `pxctl service pool expand` command specifying the following:
 
+     * UUID: The UUID of the pool you want to expand. This is found the **POOL** column when you listed the pools.
+     * SIZE: The minimum new required size of the storage pool in GiB.
+     * OPERATION-TYPE: The type of disk operation that will be performed to expand the pool. You have 2 options 
+         * **resize-disk**: Portworx will resize existing VMDKs in the storage pool. This is the recommended operation as this does not require data movement on the backing drives in the pool.
+         * **add-disk**: Portworx will create one or more VMDKs based on the required new size of the pool. After the drive is added to the pool, Portworx will rebalance the volumes onto the new VMDKs. Hence this operation can take a while.
```
Comment (nathan-px) [inline content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md]:
```suggestion
         * **add-disk**: Portworx will create one or more VMDKs based on the required new size of the pool. After the drive is added to the pool, Portworx will rebalance the volumes onto the new VMDKs. As a result, this operation can take a while to complete.
```

**Pull request merged**